src/tools/miri/tests/pass/tree_borrows/vec_unique.uniq.stderr

```diff
@@ -2,7 +2,9 @@
 Warning: this tree is indicative only. Some tags may have been hidden.
 0..   2
 | Act |    └─┬──<TAG=root of the allocation>
-|-----|      └─┬──<TAG=base.as_ptr(), base.as_ptr()>
-|-----|        └─┬──<TAG=raw_parts.0>
-|-----|          └────<TAG=reconstructed.as_ptr(), reconstructed.as_ptr()>
+|-----|      ├────<TAG=base.as_ptr()>
+|-----|      ├────<TAG=base.as_ptr()>
+|-----|      └─┬──<TAG=raw_parts.0>
+|-----|        ├────<TAG=reconstructed.as_ptr()>
+|-----|        └────<TAG=reconstructed.as_ptr()>
```

Comment:
This looks like it is now exactly doing what we do *not* want it to do. Maybe the line [here](https://github.com/rust-lang/rust/blob/1620865d7885dde9f0862ebe1c101c3f3ad461ee/src/tools/miri/tests/pass/tree_borrows/vec_unique.rs#L46) needs updating. But OTOH the entire `Unique` thing is highly experimental so you don't have to fix this here... but please add a FIXME in the test then since I am fairly certain it no longer tests what it should test with this change.

Reply:
> Maybe the line [here](https://github.com/rust-lang/rust/blob/1620865d7885dde9f0862ebe1c101c3f3ad461ee/src/tools/miri/tests/pass/tree_borrows/vec_unique.rs#L46) needs updating.

I tried bumping that depth from 2 to 3 and re-blessing, but the diff didn't change. I'll add a FIXME.